yourls/content.md

```diff
@@ -0,0 +1,56 @@
+# What is YOURLS?
+
+YOURLS is a set of PHP scripts that will allow you to run Your Own URL Shortener. You'll have full control over your data, detailed stats, analytics, plugins, and more. It's free.
+
+> [github.com/YOURLS/YOURLS](https://github.com/YOURLS/YOURLS)
+
+%%LOGO%%
+
+# How to use this image
+
+```console
+$ docker run --name some-%%REPO%% --link some-mysql:mysql -d %%IMAGE%% \
+    -e YOURLS_SITE="https://example.com"
+```
+
+The following environment variables are also honored for configuring your YOURLS instance:
+
+-	`-e YOURLS_DB_HOST=...` (defaults to the IP and port of the linked `mysql` container)
+-	`-e YOURLS_DB_USER=...` (defaults to "root")
+-	`-e YOURLS_DB_PASS=...` (defaults to the value of the `MYSQL_ROOT_PASSWORD` environment variable from the linked `mysql` container)
+-	`-e YOURLS_DB_NAME=...` (defaults to "yourls")
+-	`-e YOURLS_TABLE_PREFIX=...` (defaults to "", only set this when you need to override the default table prefix in wp-config.php)
+-	`-e YOURLS_COOKIEKEY=...` (default to unique random SHA1s)
+-	`-e YOURLS_SITE=...` (yourls instance url)
+
+If the `YOURLS_DB_NAME` specified does not already exist on the given MySQL server, it will be created automatically upon startup of the `yourls` container, provided that the `YOURLS_DB_USER` specified has the necessary permissions to create it.
+
+If you'd like to be able to access the instance from the host without the container's IP, standard port mappings can be used:
+
+```console
+$ docker run --name some-%%REPO%% --link some-mysql:mysql -p 8080:80 -d %%IMAGE%%
+```
+
+Then, access it via `http://localhost:8080` or `http://host-ip:8080` in a browser.
+
+If you'd like to use an external database instead of a linked `mysql` container, specify the hostname and port with `YOURLS_DB_HOST` along with the password in `YOURLS_DB_PASS` and the username in `YOURLS_DB_USER` (if it is something other than `root`):
+
+```console
+$ docker run --name some-%%REPO%%s -e YOURLS_DB_HOST=10.1.2.3:3306 \
+    -e YOURLS_DB_USER=... -e YOURLS_DB_PASS=... -d %%REPO%%
```

Comment:
`%%REPO%%` here at the end should be `%%IMAGE%%` instead (so the multi-arch descriptions have the right `ARCH/yourls` image name here)

Reply:
You seem right. I took WordPress template as example: https://github.com/docker-library/docs/blob/8117c12eda4829c1c57eb07c02a62028d1e2bfe8/wordpress/content.md